citesphere/src/main/webapp/WEB-INF/views/admin/apps/details.jsp

```diff
@@ -42,3 +69,20 @@
 	<td>${applicationType}</td>
 </tr>
 </table>
+<div class="modal fade" id="client-update-confirmation" tabindex="-1" role="dialog">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title">Regenerate Client Secret</h4>
+      </div>
+      <div class="modal-body">
+        <p>Once regenerated, Applications using this client id and secret won't be able to connect anymore. Are you sure you want to regenerate secret?</p>
```

Comment:
Let's say "Once a new secret has been generated, **a**pplications using the old secret..."